.github/workflows/ci.yml

```diff
@@ -16,51 +14,24 @@ permissions:
 
 jobs:
   test:
-    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        # See <https://julialang-s3.julialang.org/bin/versions.json> for available Julia versions
-        include:
-          - arch: "x64"
-            os: "ubuntu-latest"
-            version: "1.10"
-          - arch: "x64"
-            os: "windows-latest"
-            version: "1.10"
-          - arch: "x64"
-            os: "macOS-latest"
-            version: "1.10"
-          - arch: "x86"
-            os: "ubuntu-latest"
-            version: "1.10"
-          - arch: "x64"
-            os: "ubuntu-latest"
-            version: "1.9"
-          - arch: "x64"
-            os: "ubuntu-latest"
-            version: "1.8"
-          - arch: "x64"
-            os: "ubuntu-latest"
-            version: "1.7"
-          - arch: "x64"
-            os: "ubuntu-latest"
-            version: "1.6"
-          # - arch: "x64"
-          #   os: "ubuntu-latest"
-          #   version: "nightly"
+        julia-version: ['lts', '1']
```

Comment:
I think it's good to be explicit about version numbers since SIMD.jl has some low-level code. I'd write `'1.10, 1.11` here.